colossus/src/main/scala/colossus/controller/Controller.scala

```diff
@@ -16,7 +18,8 @@ trait StreamMessage {
 }
 
 case class ControllerConfig(
-  outputBufferSize: Int
+  outputBufferSize: Int,
```

Comment:
We should really doc this class.  It's not immediately obvious what these values do.